background/providers/rss/lib/parse-feed.js

```diff
@@ -0,0 +1,120 @@
+import {EventEmitter} from 'events';
+import {promisifyAll} from 'bluebird';
+import xml2js from 'xml2js';
+
+const {parseStringAsync} = promisifyAll(xml2js);
```

Comment:
If we only have to worry about the latest version of node we can switch to using [util.promisify](https://nodejs.org/en/blog/release/v8.0.0/#improved-support-for-promises) which would look something like this (basically the same but without the bluebird dependency):

```javascript
import {promisify} from 'util';
import {someFunc} from 'someModule';

const someFuncAsync = promisify(someFunc);
```

Other than that I'm open to using any other library.